docs/dictionary/property/minStackFileVersion.lcdoc

```diff
@@ -0,0 +1,36 @@
+Name: minStackFileVersion
+
+Type: property
+
+Syntax: get the minStackFileVersion of <stack>
+
+Summary:
+Get the minimum <stackFileVersion}stack file version> for a stack that
+will allow it to be safely <save|saved> without data loss.
+
+Associations: stack
+
+Introduced: 1.0
+
+OS: mac, windows, linux, ios, android, html5
+
+Platforms: desktop, server, mobile
+
+Example:
+set the stackFileVersion to the minStackFileVersion of this stack
```

Comment:
`stackFileVersion` is deprecated I think, so it would be better to use
```
save this stack with format (the minStackFileVersion of this stack)
```
as the example